ADOPTERS.md

```diff
@@ -0,0 +1,36 @@
+| -------------------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| [Alibaba](https://www.alibaba.com)     | [@johnlanni](https://github.com/johnlanni)                   | ![production](https://img.shields.io/badge/-production-blue?style=flat) | Higress is a next-generation cloud-native gateway built on the core of open-source Istio + Envoy based on Alibaba's internal Envoy Gateway practice. |
+| [Amazon](https://www.amazon.com)       | [@loganek](https://github.com/loganek)                       | ![production](https://img.shields.io/badge/-production-blue?style=flat) | Amazon Prime Video is an American subscription video on-demand over-the-top streaming and rental service of Amazon offered as a standalone service or as part of Amazon's Prime subscription. |
+| [Ant Group](https://www.antgroup.com)  | [@wei-tang](https://github.com/wei-tang)                     | ![production](https://img.shields.io/badge/-production-blue?style=flat) | AntChain is a blockchain technology platform owned by Ant Group, a Chinese tech conglomerate that also runs Alipay, China's largest digital payment system. |
+| [Bosch](https://www.bosch.com)         | emily.ruppel@us.bosch.com | ![Undisclosed](https://img.shields.io/badge/-Undisclosed-blue?style=flat)   | Silverline Cloud-Edge platform                              |
+| [Disney](https://www.disney.com)     |                                                 | ![production](https://img.shields.io/badge/-production-blue?style=flat)                                                 | Disney+ Streaming                                            |
+| [Intel](https://www.intel.com)      | [@wenyongh](https://github.com/wenyongh)                                    | ![Undisclosed](https://img.shields.io/badge/-Undisclosed-blue?style=flat) | Edge and the embedded environments |
+| [Moonbit](https://www.moonbitlang.com) | [@peter-jerry-ye](https://github.com/peter-jerry-ye) | ![Undisclosed](https://img.shields.io/badge/-Undisclosed-blue?style=flat) | MoonBit is an end-to-end programming language toolchain for cloud and edge computing using WebAssembly. |
+| [Microsoft](https://www.microsoft.com) | [@Mossaka](https://github.com/Mossaka)                       | ![production](https://img.shields.io/badge/-production-blue?style=flat)                                                       | Hyperlight runs Wasm workloads in VMs without OS and kernel, it is a solution for improving the management and security of Wasm workloads on Azure. |
+| [Midokura](https://www.midokura.com)                          |   [@yamt](https://github.com/yamt)                                     | ![Undisclosed](https://img.shields.io/badge/-Undisclosed-blue?style=flat) | WEdge: the next-generation Edge AI sensing platform                                                   |
```

Comment:
it seems that the name "WEdge" is no longer used.
i'm not sure what's the latest name.
please just remove "WEdge: " for now.

Reply:
Done.